_site/authors/etagwerker/index.html

```diff
@@ -115,7 +115,7 @@ <h2 class="post-title"><a class="read-more" href="/blog/software%20development/s
     <script type="text/javascript">
 
       var _gaq = _gaq || [];
-      _gaq.push(['_setAccount', '<%= ENV['GOOGLE_ANALYTICS_ID'] %>']);
+      _gaq.push(['_setAccount', '']);
```

Comment:
@etagwerker Looks like it's not populating the ID for `authors/:author/index.html`.